proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -262,6 +262,7 @@ message Message {
   string file_path = 5;
   bool is_map_entry = 6;
   repeated Field fields = 7;
+  repeated OneOf oneofs = 10;
```

Comment:
This should be moved after the 9th field number (so that they're in order).

Reply:
I guess I wanted this logically to be grouped next to `fields`, but also didn't want to swap the field numbers because I didn't want to make an unnecessary breaking change. Thoughts on what the best way to show this is? (Also, thanks for the stamp, haha, I'm just wondering if we have any thoughts on the style for this)

Reply:
Not a huge deal either way - I prefer field numbers to be in order since that's what we do elsewhere, but that's just ne.

Reply:
Sounds good, not too worried about it, was just wondering because I've seen different styles for doing this (e.g. descriptor.proto has some random ordering things https://github.com/protocolbuffers/protobuf/blob/master/src/google/protobuf/descriptor.proto#L66-L72).